javascript/stimulus_reflex.js

```diff
@@ -239,4 +239,10 @@ document.addEventListener('cable-ready:after-inner-html', afterDOMUpdate)
 document.addEventListener('cable-ready:after-morph', afterDOMUpdate)
 window.addEventListener('load', setupDeclarativeReflexes)
```

Comment:
we might even be able to  remove the 
```js
window.addEventListener('load', setupDeclarativeReflexes)
```

since  `readystatechange` could be enough in all cases. But I can't recall why we are using `load` on window here.

Reply:
It could just be that it's always been this way. If you feel confident that the `readystatechange` approach is rock solid, I'd be perfectly fine with you swapping out.

Reply:
We should be good with just the `readystatechange` EventListener. `window.onload` is almost identical to `document.readyState === 'complete'` and is fired almost immediately after `window.onload`, see: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event